cove/templates/base_generic.html

```diff
@@ -73,11 +73,13 @@
 
   {% block howToUse %}
   {% endblock %}
-  
+
+  {% block moreInfo %}
   <div id="more-information" class="panel panel-default">
     <div class="panel-heading" data-toggle="collapse" data-target="#more-information-body">
       <h3 class="panel-title"> {% blocktrans %}More Information {% endblocktrans %} <small> {% blocktrans %}click to see.{% endblocktrans %} </small></h3>
     </div>
+  {% endblock %}
```

Comment:
This `endblock` is in the wrong place. It should be after the end of the #more-information div, which is on line 104/106.

As a result the current html is invalid https://validator.w3.org/nu/?doc=http%3A%2F%2F553-more-information-box-is-blank.dev.cove.opendataservices.coop%2F360%2Fcommon_errors
Our automated tests didn't catch this because the commen errors page isn't listed, maybe it should be https://github.com/OpenDataServices/cove/blob/master/.travis.yml#L22